modules/bbp-electrophysiology/src/main/resources/schemas/bbp/electrophysiology/stimulusexperiment/v0.1.0.json

```diff
@@ -0,0 +1,90 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+    "bbpactsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+    "bbptlotsh" : "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+    "bbpqsh" : "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+    "this" : "{{base}}/schemas/bbp/electrophysiology/stimulusexperiment/v0.1.0/shapes/"
+    }
+    ],
+  "@type" : "owl:Ontology",
+  "imports" : [
+                "{{base}}/schemas/bbp/core/activity/v0.1.0",
+                "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"
+    ],
+  "shapes" : [
+    {
+      "@id" : "this:StimulusExperimentShape",
+      "@type" : "sh:NodeShape",
+      "label" : "Stimulus experiment shape definition",
+      "targetClass" : "nsg:StimulusExperiment",
+      "nodeKind" : "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "bbpactsh:ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:providerExperimentId",
+              "name": "Provider experiment ID",
+              "description": "Laboratory identifier of the experiment",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:providerExperimentName",
+              "name": "Provider experiment name",
+              "description": "Laboratory name of the experiment",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path" : "nsg:stimulus",
+              "name" : "Stimulus",
+              "description" : "The shape of the stimulus",
+              "node" : "this:StimulusShape",
+              "minCount" : 1,
+              "maxCount" : 1
+            },
+            {
+              "path": "prov:used",
+              "description": "The patched cell used",
+              "class": "nsg:PatchedCell",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "this:StimulusShape",
+      "@type": "sh:NodeShape",
+      "label" : "Stimulus shape definition",
+      "property": [
+        {
+          "path": "nsg:stimulus",
```

Comment:
I would suggest to find another name for the property which will hold the applied stimulus to avoid repetition with the parent property:
```json
 "stimulus": {
       "stimulus": {....}
    ...
}
```

What about stimulusValue ?

Reply:
Good point! I would maybe suggest "stimulusType" instead?

Reply:
excellent.